### PR TITLE
update fr docs concepts overview

### PR DESCRIPTION
**Pull request by rekcah78**

### CODE REVIEW

File: content/fr/docs/concepts/overview/working-with-objects/_index.md
Comment (perriea) [inline content/fr/docs/concepts/overview/working-with-objects/_index.md:1]: This part isn't translated yet

**Review by perriea — CHANGES_REQUESTED** [review]
Sounds good except two file not translated yet :)

File: content/fr/docs/concepts/overview/working-with-objects/object-management.md
[the remark this answers is not in the document]
Comment (rekcah78) [reply content/fr/docs/concepts/overview/working-with-objects/object-management.md:1]: done

File: content/fr/docs/concepts/overview/working-with-objects/_index.md
Comment (rekcah78) [reply content/fr/docs/concepts/overview/working-with-objects/_index.md:1]: done

**Comment by perriea** [comment]
Nothing more to add? @rekcah78

**Comment by rekcah78** [comment]
> Nothing more to add? @rekcah78

no more for this PR

**Pull request merged**